NotadogApi/Domain/Game/Room.cs

```diff
@@ -24,16 +24,19 @@ public class Room
     {
         public Guid Guid { get; }
         public List<User> Players { get; }
+        public List<User> MakedMovePlayers { get; }
         public int PlayersMaxCount { get; }
```

Comment:
I suggest to rename PlayersMaxCount to something like PlayersToStart. Also better make it nullable (int?) then assign 0 value for private rooms.

Reply:
mm it's not players to start) it's max players count. And now it works as expected. It's 0 by default, and it can be setted to any number from user.